### ISSUE TITLE
swag cant install

### ISSUE THREAD

**Issue by bkocadere** [swaggo/swag#610]
I cant install swag
![image](https://user-images.githubusercontent.com/26333526/72208853-05da2d00-34b9-11ea-90de-88ab578d2270.png)

**Your swag version**
last
**Your go version**
1.13.6 windows/amd64

**Comment by Graphmaxer** [comment]
Same here, for a workaround I use the binary of this release https://github.com/swaggo/swag/releases/tag/v1.6.4

**Comment by bkocadere** [comment]
> Same here, for a workaround I use the binary of this release https://github.com/swaggo/swag/releases/tag/v1.6.4

thanks, how can i install binary file?

**Comment by Graphmaxer** [comment]
> > Same here, for a workaround I use the binary of this release https://github.com/swaggo/swag/releases/tag/v1.6.4
> 
> thanks, how can i install binary file?

A better solution for your is just to use this command: `go get -u github.com/swaggo/swag/cmd/swag@v1.6.3`

In my case I was using a dockerfile and I just needed the binary, so I used wget to download it and I used it.

**Comment by sdghchj** [comment]
v1.6.4 or v1.6.5 ?

**Comment by Graphmaxer** [comment]
> v1.6.4 or v1.6.5 ?

Last one, v1.6.5, also it is the same error as this old issue https://github.com/swaggo/swag/issues/556, but the fix got reverted recently.

**Comment by sdghchj** [comment]
@easonlin404

**Comment by bkocadere** [comment]
1.6.5

**Comment by easonlin404** [comment]
@bkocadere I think it caused by `GO111MODULE=auto`, can you execute:
```
export GO111MODULE=on
```
and try again
```
go get -u github.com/swaggo/swag/cmd/swag
```

**Comment by bkocadere** [comment]
> @bkocadere I think it caused by `GO111MODULE=auto`, can you execute:
> 
> ```
> export GO111MODULE=on
> ```
> 
> and try again
> 
> ```
> go get -u github.com/swaggo/swag/cmd/swag
> ```

thanks, how can use 'export' in cmd screen ?

**Comment by easonlin404** [comment]
sorry, I didn't note that you use windows environment, please refer https://superuser.com/questions/79612/setting-and-getting-windows-environment-variables-from-the-command-prompt/79614 to set environment variables like:
```
SET GO111MODULE=on
```

**Comment by bkocadere** [comment]
> sorry, I didn't note that you use windows environment, please refer https://superuser.com/questions/79612/setting-and-getting-windows-environment-variables-from-the-command-prompt/79614 to set environment variables like:
> 
> ```
> SET GO111MODULE=on
> ```

thanks , working

**Comment by Graphmaxer** [comment]
Yo should close this issue, the bug is still here.

**Comment by alvioralali** [comment]
> @bkocadere I think it caused by `GO111MODULE=auto`, can you execute:
> 
> ```
> export GO111MODULE=on
> ```
> 
> and try again
> 
> ```
> go get -u github.com/swaggo/swag/cmd/swag
> ```

thanks bro it's working

**Comment by easonlin404** [comment]
Please use the latest version against master to try again, the issue shoud be fixed by #618.

**Comment by leofernandezandreani** [comment]
> @bkocadere I think it caused by `GO111MODULE=auto`, can you execute:
> 
> ```
> export GO111MODULE=on
> ```
> 
> and try again
> 
> ```
> go get -u github.com/swaggo/swag/cmd/swag
> ```

Thanks! Its working. I had days with this problem
